### DISCUSSION TITLE
Welcome to RSpotify Discussions!

### DISCUSSION THREAD

**Discussion by ramsayleung** [ramsayleung/rspotify#340] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marioortizmanero** [comment]
Yup, recently I've been seeing lots of issues with questions. They are completely fine, but I think that moving them here will help tidy everything up. I'm sure it will be useful to brainstorm + similars as well.

If anyone doesn't like this format please let us know, I'm just seeing it as a way to experiment. I've never tried GitHub discussions and they seem to be worth a try.
